tests/infra/interfaces.py

```diff
@@ -96,6 +106,8 @@ def from_json(json):
         )
         if "endorsement" in json:
             interface.endorsement = Endorsement.from_json(json["endorsement"])
+        if "acme_configuration" in json:
+            interface.acme_configuration = json.get("acme_configuration")
```

Comment:
get() returns None if the key is missing, so you probably don't need the condition here (it's used above because getitem will raise if the key is missing).